pkg/server/container_create_windows.go

```diff
@@ -166,8 +171,28 @@ func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateConta
 
 	log.G(ctx).Debugf("Container %q spec: %#+v", id, spew.NewFormatter(spec))
 
-	snapshotterOpt := snapshots.WithLabels(config.Annotations)
+	// If the config field is specified, set the snapshotter label to reuse the pods
+	// scratch space. This only affects LCOW at the moment.
+	val, ok := config.Annotations["containerd.io/snapshot/io.microsoft.container.storage.reuse-scratch"]
+	if rhcso.ShareScratch || (ok && val == "true") {
+		config.Annotations["containerd.io/snapshot/io.microsoft.container.storage.reuse-scratch"] = "true"
+		_, ok := config.Annotations["containerd.io/snapshot/io.microsoft.owner.key"]
+		if !ok {
+			config.Annotations["containerd.io/snapshot/io.microsoft.owner.key"] = sandboxID
+		}
+	}
 
+	if rhcso.DefaultContainerScratchSizeInGb != 0 {
+		size := strconv.FormatInt(int64(rhcso.DefaultContainerScratchSizeInGb), 10)
+		_, ok = config.Annotations["containerd.io/snapshot/io.microsoft.container.storage.rootfs.size-gb"]
+		// If the client wanted to override the default set this should be possible by specifying the annotation
+		// on a per container basis.
+		if !ok {
+			config.Annotations["containerd.io/snapshot/io.microsoft.container.storage.rootfs.size-gb"] = size
```

Comment:
Should we log here that we're overriding the `DefaultContainerScratchSizeInGb`?

Reply:
Sure